<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateaccount</class>
 <widget class="QMainWindow" name="updateaccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>401</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 2px solid rgb(16, 16, 16);
    border-radius: 15px;
    margin-top: 10px; 
    background-color:rgb(232, 232, 232);
    padding: 5px;
 
}

</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Update your Account information here</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Phone No.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Email ID</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Grade</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Reg No.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEdit_updtname">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color: rgb(255, 255, 255,0);
    min-width: 8px; 
    min-height: 8px;
    padding: 5px;
    margin: 0px;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_updtphone">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color: rgb(255, 255, 255,0);
    min-width: 8px; 
    min-height: 8px;
    padding: 5px;
    margin: 0px;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_updtemail">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color: rgb(255, 255, 255,0);
    min-width: 8px; 
    min-height: 8px;
    padding: 5px;
    margin: 0px;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_updtpassword">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color: rgb(255, 255, 255,0);
    min-width: 8px; 
    min-height: 8px;
    padding: 5px;
    margin: 0px;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_updtgrade">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color: rgb(255, 255, 255,0);
    min-width: 8px; 
    min-height: 8px;
    padding: 5px;
    margin: 0px;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_updtregno">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color: rgb(255, 255, 255,0);
    min-width: 8px; 
    min-height: 8px;
    padding: 5px;
    margin: 0px;
}
</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_update">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color:rgb(255, 255, 255,0);
    min-width: 9px; 
    min-height: 9px; 
    padding: 5px; 
    margin: 0px; 
}
</string>
         </property>
         <property name="text">
          <string>Update </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_home">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid rgb(16, 16, 16);
    border-radius: 10px; 
    background-color:rgb(255, 255, 255,0);
    min-width: 9px; 
    min-height: 9px; 
    padding: 5px; 
    margin: 0px; 
}
</string>
       </property>
       <property name="text">
        <string>Back to Home</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
